cli/packages/cmd/secrets.go

```
@@ -143,7 +143,12 @@ var secretsSetCmd = &cobra.Command{
 	Short:                 "Used set secrets",
 	Use:                   "set [secrets]",
 	DisableFlagsInUseLine: true,
-	Args:                  cobra.MinimumNArgs(1),
+	Args: func(cmd *cobra.Command, args []string) error {
+		if cmd.Flags().Changed("file") {
+			return cobra.ExactArgs(0)(cmd, args)
```

Please throw an error if the user is trying to pass command-line arguments and the `file` flag at the same time, just so it's a little more clear why it's failing